### ISSUE TITLE
On rendering the mano.

### ISSUE THREAD

**Issue by aragakiyui611** [facebookresearch/InterWild#3]
https://github.com/facebookresearch/InterWild/blob/12f68107686d2ad52a151506880c7e3e5f7d0e88/data/InterHand26M/InterHand26M.py#L52

I encountered an exception that missing the file `aid_human_annot_*.txt` in the InterHand26M.py, could you provide these files? Thank you!

**Comment by mks0601** [comment]
Hi, could you check this? Thanks
https://github.com/facebookresearch/InterWild#start

**Comment by aragakiyui611** [comment]
Thank you every much

**Comment by aragakiyui611** [comment]
Hi, there is a point that I am not very understand:
https://github.com/facebookresearch/InterWild/blob/f8b753144b9d5230c8136697172efd53f153f35d/main/model.py#L175
this loss supervises the joint_proj under output_hand_hm_shape space, but the labels in MSCOCO: https://github.com/facebookresearch/InterWild/blob/f8b753144b9d5230c8136697172efd53f153f35d/data/MSCOCO/MSCOCO.py#L169
uses this function which output joint_img under output_body_hm_shape space:
https://github.com/facebookresearch/InterWild/blob/f8b753144b9d5230c8136697172efd53f153f35d/common/utils/preprocessing.py#L178

**Comment by mks0601** [comment]
So my plan is changing TransNet to additionally output root_cam as it takes two-hand input so that the additional root_cam could make correspondence between two-hand meshes and two-hand 2D inputs. Using that, we could render 3D meshes on images.

**Comment by mks0601** [comment]
yeap img2bb_trans should be a one reason, but fundamentally, I think additional root_cam is necessary as I tried the exact same thing as yours. Please let me know if you have some results :)

**Comment by mks0601** [comment]
As you said, using the rroot_cam is not strictly accurate, and projecting 3D two-hand meshes is not possible at this moment. If you're in a hurry, your approximation could be a hacky way for your purpose? I guess one alternative way is to iterative optimize rroot_cam per frame, which would take much time

**Comment by mks0601** [comment]
Yeap. for the single hand pck, you could just use them separately.

**Comment by txytju** [comment]
> https://github.com/facebookresearch/InterWild/blob/12f68107686d2ad52a151506880c7e3e5f7d0e88/data/InterHand26M/InterHand26M.py#L52
> 
> I encountered an exception that missing the file `aid_human_annot_*.txt` in the InterHand26M.py, could you provide these files? Thank you!

@aragakiyui611 同关注这个方向，可以加个微信不哈哈！我微信 txy13512216389

**Comment by mks0601** [comment]
Hi, visualization feature is added in here
https://github.com/facebookresearch/InterWild/blob/main/demo/demo.py

**Comment by aragakiyui611** [comment]
@mks0601 Hi I found that in the dataloader 
https://github.com/facebookresearch/InterWild/blob/25402806951f273353b0fd2a446346fa7cea5815/common/utils/preprocessing.py#L202
https://github.com/facebookresearch/InterWild/blob/25402806951f273353b0fd2a446346fa7cea5815/common/utils/preprocessing.py#L267 which gives joint_img under output_body_hm_shape, but the model outputs joint_img under output_hand_hm_shape. 
https://github.com/facebookresearch/InterWild/blob/25402806951f273353b0fd2a446346fa7cea5815/common/nets/module.py#L20
Is that correct or something I missed?

**Comment by mks0601** [comment]
You're correct, and that is why I change cfg.output_body_hm_shape to cfg.output_hand_hm_shape in here
https://github.com/facebookresearch/InterWild/blob/25402806951f273353b0fd2a446346fa7cea5815/main/model.py#L129
